### ISSUE TITLE
parsed value has first letter capital

### ISSUE THREAD

**Issue by abhi5658** [motdotla/dotenv#462]
I created my `.env` file which has a line like this:
`username=abhishek`

When calling this environment variable inside file(let's say `console.log(process.env.username)` the output is Abhishek i.e. the value with first letter capital.
Until now this only happened with the variable name `username` but not any other variable. Like I had to use `user=abhishek` for time being.
Is it a bug or any incorrect use of .env file?

**Comment by bijaydas** [comment]
I just tried with the example. But I am not getting such output. Which Node version are you using?

**Comment by abhi5658** [comment]
node version : `12.16.1`
npm version : `6.13.4`
dotenv version : `^8.2.0`

.env file : 
![image](https://user-images.githubusercontent.com/17237508/77905862-9c421180-72a4-11ea-84e8-ba23545b3ac0.png)
File : 
![image](https://user-images.githubusercontent.com/17237508/77905732-6ac94600-72a4-11ea-8344-f051a192b95d.png)
Output : 
![image](https://user-images.githubusercontent.com/17237508/77905904-ab28c400-72a4-11ea-8cc7-5556bf128850.png)

What can be the issue?
As I mentioned earlier I am facing this issue only with the `username` variable and not with any other variable like the `user`. And I am using vscode on Windows 10.

**Comment by ChristianUlbrich** [comment]
I bet your `process.env.username` is or has already been set **before** `dotenv` does itzzzz magic. Try running your script, **without** _requiring_ `dotenv`.
`dotenv` does not modify _existing_ env vars. -> https://github.com/motdotla/dotenv#what-happens-to-environment-variables-that-were-already-set

**Comment by abhi5658** [comment]
Found it. Its coming from the pre-existing default system environment variables where the username is  set to the logged in user. As yes mentioned by Christian, it does not get override.
Also, I set the variable to my name(in lowercase: 'abhishek') but the default system environment has username 'Abhishek' (first letter capital: issue) and this was not overidden.

It was such a small defaulter and created so much of a issue.

I  would suggest others to create variable names that do not exist by default  when launching an application. Only when the default  values do matter, use the default variable name.
